helpers/utils

```diff
@@ -172,6 +175,10 @@ ynh_setup_source() {
         done
     fi
 
+    if [ "$full_replace" -eq 1 ]; then
+        ynh_secure_remove --file="$dest_dir"
```

Comment:
Should we test if $dest_dir is a mount point ?

If we delete the mount point and recreate the dir, the sources could be installed on the bad disk/partition.

Reply:
I suppose, but that doesn't really look like a regression from this specific commit.

We have similar behavior when eg. failed upgrade will trigger a remove + backup restore, and the remove itself also include an `ynh_secure_remove` ...